### ISSUE TITLE
Error: Cannot find module 'handlebars/handlebars.runtime.js'

### ISSUE THREAD

**Issue by mszymulanski** [justusromijn/gulp-precompile-handlebars#1]
Handlebars [node module](https://www.npmjs.com/package/handlebars) doesn't have `handlebars.runtime.js` file. There seems to be `runtime.js`. 

Changing [this line](https://github.com/justusromijn/gulp-precompile-handlebars/blob/master/index.js#L13) seems to fix the issue:
From
```
Handlebars: 'handlebars/handlebars.runtime.js'
```
To
```
Handlebars: 'handlebars/runtime.js'
```

However, the unit test must be fixed too.

**Comment by justusromijn** [comment]
I think I will use the file from the lib directory: https://github.com/wycats/handlebars.js/tree/master/lib

I used jspm to reference handlebars in my own project, hence the link worked there as I had a named prefix.

**Comment by justusromijn** [comment]
Should be fixed with release 2.0.6
